### ISSUE TITLE
Same COLLADA files with different names in separate packages

### ISSUE THREAD

**Issue by 130s** [tork-a/rtmros_nextage#68]
tldr; I don't think we need to change anything regarding this(, at least for now).

[nextage_ros_bridge/models/nextage.dae](https://github.com/tork-a/rtmros_nextage/blob/795a41e678c9999c20086c4409d686f2fefa1bb0/nextage_ros_bridge/models/nextage.dae) and [nextage_description/models/main.dae](https://github.com/tork-a/rtmros_nextage/blob/795a41e678c9999c20086c4409d686f2fefa1bb0/nextage_description/models/main.dae) look the same. 

Even [in its tutorial](http://wiki.ros.org/rtmros_nextage/Tutorials/Changing%20Nextage%20Grippers%20in%20Robot%20Model#Create_COLLADA_model) the file is copied from one to another. That said `nextage_ros_bridge/models/nextage.dae` may seem redundant.

```
 $ roscd nextage_description/models
 $ rosrun openhrp3 export-collada -i main.wrl -o main.dae
 $ cp main.dae ../../nextage_ros_bridge/nextage.dae
```

I assume this setting is necessary due to how the config files are auto-generated (ie. in order to remove `nextage_ros_bridge/models/nextage.dae` we might have to change upstream package like `hrpsys_ros_robotics`)?

```
rtmros_nextage$ grep -r nextage.dae .
Binary file ./.git/index matches
./nextage_ros_bridge/launch/nextage_ros_bridge_viz.launch:  <arg name="COLLADA_FILE" default="$(find nextage_ros_bridge)/models/nextage.dae" />
./nextage_ros_bridge/launch/nextage_ros_bridge.launch:  <arg name="COLLADA_FILE" default="$(find nextage_ros_bridge)/models/nextage.dae" />
./nextage_ros_bridge/doc/tutorial/changing_nextage_grippers_in_robot_model.wiki: $ cp main.dae ../../nextage_ros_bridge/nextage.dae
./nextage_ros_bridge/conf/nextage_nosim.xml:  <item class="com.generalrobotix.ui.item.GrxModelItem" name="nextage" url="/home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae">
./nextage_ros_bridge/conf/nextage_nosim.RobotHardware.conf:model: file:///home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
./nextage_ros_bridge/conf/nextage.xml:  <item class="com.generalrobotix.ui.item.GrxModelItem" name="nextage" url="/home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae">
./nextage_ros_bridge/conf/nextage.conf:model: file:///home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
./nextage_ros_bridge/conf/nextage_nosim.conf:model: file:///home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
./nextage_ros_bridge/conf/nextage.RobotHardware.conf:model: file:///home/n130s/data/Dropbox/ROS/hydro_precise/cws_hiro_nxo/src/tork-a/rtmros_nextage/nextage_ros_bridge/models/nextage.dae
```

**Comment by k-okada** [comment]
I forget why we add two files, maybe we can remove main.dae and 

```
$ rosrun openhrp3 export-collada -i main.wrl -o main.dae
$ cp main.dae ../../nextage_ros_bridge/nextage.dae
```

can be replaced by

```
$ rosrun openhrp3 export-collada -i main.wrl -o  ../../nextage_ros_bridge/nextage.dae
```

???? but not sure.

**Comment by 130s** [comment]
Or I feel even more comfortable to gather model files in `nextage_description` and remove `models` directory from `nextage_ros_bridge`. Is it not feasible?

**Comment by k-okada** [comment]
Yes, I think is is possible.
One problem of current `nextage_description` is there are no information on how to convert WRL to collada/urdf and nextage_ros_bridge have them (just for collada). So desirable procedure is
1) update nextage_description/CMakeLists.txt to convert wrl to urdf and dae and commit them
2) remove model file from nextage_ros_bridge.

**Comment by 130s** [comment]
> One problem of current nextage_description is there are no information on how to convert WRL to collada/urdf and nextage_ros_bridge have them (just for collada). So desirable procedure is
> 1) update nextage_description/CMakeLists.txt to convert wrl to urdf and dae and commit them
> 2) remove model file from nextage_ros_bridge. 

(2) should be done in https://github.com/tork-a/rtmros_nextage/pull/136. 

For (1) I tried (it can be found in https://github.com/130s/rtmros_nextage/commit/867128392f3e34f2526e5545fdf5f360da1544c2) but haven't got it working so far.

**Comment by k-okada** [comment]
> (it can be found in 130s@8671283) 

seems ok for me, see https://github.com/fkanehiro/openhrp3/blob/master/sample/CMakeLists.txt for 

## working version. Please create PR to see error message on travis.

◉ Kei Okada
